src/rules/functionality.ts

```diff
@@ -51,6 +59,10 @@ export default {
   'no-string-throw': true,
   // Disallows falling through case statements.
   'no-switch-case-fall-through': true,
+   // Warns when a method is used as outside of a method call.
+  'no-unbound-method': false,
```

Comment:
Leave it off.  We use a mix of `@autobind` and constructor binding.  

We could potentially write our own rule that checks for autobind/constructor-binding before flagging any unbound method refs.